nixos/modules/services/web-apps/snipe-it.nix

```diff
@@ -309,12 +298,12 @@ in {
     services.snipe-it.config = {
       APP_ENV = "production";
       APP_KEY._secret = cfg.appKeyFile;
+      APP_KEY.prefix = "base64:";
```

Comment:
I guess it's a small breaking change that `base64:` is alway added. The module used to only add it if it wasn't there.